### DISCUSSION TITLE
in HTML5, new_data_update/patch signal isn't emitted

### DISCUSSION THREAD

**Discussion by boseagr** [GodotNuts/GodotFirebase#178] [Q&A] [answered]
Hello,
Thanks for your great work. I just try your FirebaseDemo-Chat and export it to HTML5. I am using Godot 3.2.2 however I can't get the chat to appear on the screen, using export to windows works was just fine., the chat appears on the screen after I type and send it. 

Note when I send the chat both on HTML5 and desktop, the chat send properly to my realtime database.

Is there anything I should do to make it work?

THanks

**Comment by PadreMontoya** [comment]
I think there are known issues with HTML5, as noted by the wiki.

"At the moment, there appears to be a bug with HTML5 and UWP export, but the team will be verifying/debugging it in the near future"

Could you share a link to the chat project repo, though?  I wanted to see how something like that was structured.

**Comment by WolfgangSenff** [comment, marked as answer]
This issue should be resolved in 3.3 - the fetch() API implementation by Fales (from the Godot team) resolved the issues mentioned by @PadreMontoya above, which was a great callout. I haven't tested it explicitly, but there was definitely a major bug in Godot itself preventing us from utilizing updates that was resolved.

@PadreMontoya - you can find all our repos under https://github.com/GodotNuts.
